content/chef_compliance_phase.md

```diff
@@ -0,0 +1,431 @@
++++
+title = "Compliance Phase"
+draft = false
+
+aliases = ["/chef_compliance_phase.html"]
+
+[menu]
+  [menu.infra]
+    title = "Compliance Phase"
+    identifier = "chef_infra/features/chef_compliance_phase.md Compliance Phase"
+    parent = "chef_infra/features"
+    weight = 80
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/chef_compliance_phase.md)
+
+## Overview
+
+Compliance Phase enables Chef Infra to run Chef InSpec profiles as part of a client run, by downloading profiles from sources like Chef Automate, Chef Supermarket, or GitHub, and reporting the results to Chef Automate or saving the results locally. Compliance Phase has been implemented to be mostly compatible with [audit cookbook](https://github.com/chef-cookbooks/audit), so that users of the audit cookbook can remove it from the node's runlist and get the same behavior.
+
+## Usage
+
+### Configuration
+
+Compliance Phase needs to be configured for each node where the `chef-client` runs. All node-specific configuration is done via Chef Infra attributes. Specifying one or more Chef InSpec profiles is the minimum configuration necessary to enable Compliance Phase in a Chef Infra client run. With no further configuration, this will cause a Chef Infra client run to execute the Chef InSpec profiles and write the results to disk using the default `json-file` reporter.
```

Comment:
```suggestion
Compliance Phase needs to be configured for each node where the Chef Infra Client runs. All node-specific configuration is done via Chef Infra attributes. Specifying one or more Chef InSpec profiles is the minimum configuration necessary to enable Compliance Phase in a Chef Infra Client run. With no further configuration, this will cause a Chef Infra client run to execute the Chef InSpec profiles and write the results to disk using the default `json-file` reporter.
```